src/helpers/helpers.js

```diff
@@ -0,0 +1,32 @@
+const flatten = require('flat')
```

Comment:
@jperasmus I added this new `helpers` file. Not sure if this is the right place or if I should have added these methods somewhere else?